blog/tutorial/01-index.md

```diff
@@ -209,24 +219,26 @@ Going back to `GetCmdCreatePost` in `x/blog/client/cli/tx.go`, you'll see `MsgCr
 
 After being broadcast, the messages are processed by an important part of the application, called [**handlers**](https://docs.cosmos.network/master/building-modules/handler.html).
```

Comment:
ummmm I think handlers are deprecated now, Cosmos SDK modules use Msg services instead https://docs.cosmos.network/master/building-modules/msg-services.html 

what changes do we need here?